### ISSUE TITLE
Bad clearing of duplicate tags

### ISSUE THREAD

**Issue by LudwikJaniuk** [sn-extensions/batch-manager#2]
Clearing a duplicate on a tag leaves the notes that had that tag without a tag at all. It should redirect them to the correct one, essentially performing a merge.

**Comment by mobitar** [comment]
In theory that sounds nice. But the Batch Manager doesn't understand what a note or tag is. Just deals with data types. The Batch Manager isn't perfect for every situation, but can be helpful for some.

**Comment by LudwikJaniuk** [comment]
Is there a different solution for dealing with duplicates then?

**Comment by mobitar** [comment]
The Batch Manager is currently the best solution. So if that doesn't work, manual intervention might be required. It's more important we figure out what causes unnecessary duplication with tags. My theory is that it only happens to older accounts that may have had tag relationships back when they were bi-directional (note-to-tag and tag-to-note). And when they make changes to those "legacy" tags, some sort of mixup happens.

**Comment by LudwikJaniuk** [comment]
Agreed, duplicates shouldn't happen. That said, my account is only a few months old, and I'm getting lots. Think it might have something to do with the folder extension, though I'm not sure. I tried posting on slack as well.

**Comment by agundy** [comment]
Late to the conversation but a big place this comes up is on import. I imported notes from Evernote and since there was multiple notebooks implemented with shared tags across notebooks, import created a new tag for each import. Leaving my Standard Notes install littered with tags.
